frontend/snapps/httpsnapp/src/index.ts

```diff
@@ -0,0 +1,74 @@
+import fetch from 'node-fetch';
```

Comment:
can you make sure this works in the browser and NodeJS?

Reply:
Good catch, I switched out `node-fetch` to use `axios` instead which should work in both the browser and NodeJS.